basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -1,131 +1,253 @@
+import org.jboss.dmr.ModelNode;
 import org.jboss.hal.testsuite.category.Shared;
-import org.jboss.hal.testsuite.cli.CliClient;
-import org.jboss.hal.testsuite.cli.CliClientFactory;
-import org.jboss.hal.testsuite.cli.DomainManager;
-import org.jboss.hal.testsuite.dmr.AddressTemplate;
-import org.jboss.hal.testsuite.dmr.DefaultContext;
-import org.jboss.hal.testsuite.dmr.Dispatcher;
-import org.jboss.hal.testsuite.dmr.Operation;
-import org.jboss.hal.testsuite.dmr.ResourceAddress;
-import org.jboss.hal.testsuite.finder.Application;
-import org.jboss.hal.testsuite.finder.FinderNames;
-import org.jboss.hal.testsuite.finder.FinderNavigation;
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
 import org.jboss.hal.testsuite.fragment.MetricsAreaFragment;
-import org.jboss.hal.testsuite.page.runtime.DomainRuntimeEntryPoint;
-import org.jboss.hal.testsuite.page.runtime.StandaloneRuntimeEntryPoint;
-import org.jboss.hal.testsuite.page.runtime.WebMetricsPage;
+import org.jboss.hal.testsuite.page.runtime.UndertowServerMetricsPage;
 import org.jboss.hal.testsuite.util.ConfigUtils;
-import org.jboss.hal.testsuite.util.Console;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.After;
 import org.junit.AfterClass;
+import org.junit.Assert;
 import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wildfly.extras.creaper.commands.deployments.Deploy;
+import org.wildfly.extras.creaper.commands.deployments.Undeploy;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.ReadAttributeOption;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
 
 import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
 import java.util.concurrent.TimeoutException;
 
 import static org.junit.Assert.assertEquals;
 
 /**
- * Created by pcyprian on 11.8.15.
+ * Test case for testing metrics of web (undertow) connectors.
+ *
+ * Real values are generated to see whether they are really updated in the web console.
  */
 @RunWith(Arquillian.class)
+@RunAsClient
 @Category(Shared.class)
 public class WebMetricsTestCase {
 
+    private static final Logger log = LoggerFactory.getLogger(WebMetricsTestCase.class);
+
+    private static final String DEPLOYMENT_NAME = "servlets";
```

Comment:
`metrics_generating_servlets` or may be even better `WebMetricsTestCase_servlets` ?